drainer/sync/util.go

```diff
@@ -29,11 +29,15 @@ type DBConfig struct {
 	TimeLimit     string           `toml:"time-limit" json:"time-limit"`
 	SizeLimit     string           `toml:"size-limit" json:"size-limit"`
 
-	ZKAddrs          string `toml:"zookeeper-addrs" json:"zookeeper-addrs"`
-	KafkaAddrs       string `toml:"kafka-addrs" json:"kafka-addrs"`
-	KafkaVersion     string `toml:"kafka-version" json:"kafka-version"`
-	KafkaMaxMessages int    `toml:"kafka-max-messages" json:"kafka-max-messages"`
-	TopicName        string `toml:"topic-name" json:"topic-name"`
+	ZKAddrs            string `toml:"zookeeper-addrs" json:"zookeeper-addrs"`
+	KafkaAddrs         string `toml:"kafka-addrs" json:"kafka-addrs"`
+	KafkaVersion       string `toml:"kafka-version" json:"kafka-version"`
+	KafkaMaxMessages   int    `toml:"kafka-max-messages" json:"kafka-max-messages"`
+	KafkaSASLEnable    bool   `toml:"kafka-sasl-enable" json:"kafka-sasl-enable"`
+	KafkaSASLHandshake bool   `toml:"kafka-sasl-handshake" json:"kafka-sasl-handshake"`
+	KafkaSASLUser      string `toml:"kafka-sasl-user" json:"kafka-sasl-user"`
+	KafkaSASLPassword  string `toml:"kafka-sasl-password" json:"kafka-sasl-password"`
```

Comment:
Could you please also add some documentation for these new options?

1. `cmd/drainer/drainer.toml` acts like an example configuration file
1. https://github.com/pingcap/docs-cn/blob/master/tools/binlog/tidb-binlog-kafka.md is the documentation